hw/top_earlgrey/ip_autogen/clkmgr/BUILD

```diff
@@ -6,7 +6,5 @@ package(default_visibility = ["//visibility:public"])
 
 filegroup(
     name = "all_files",
-    srcs = glob(["**"]) + [
-        "//hw/ip_templates/clkmgr/data:all_files",
-    ],
+    srcs = glob(["**"]) + ["data:all_files"],
```

Comment:
We don't usually do labels with relative paths for anything outside the current directory. Let me know if you object and would prefer we do this differently!

Reply:
Oh, I guess this is invalid. That's weird: I could've sworn I was able to write these in the past. And it builds locally /shrug